Dockerfile

```diff
@@ -208,6 +208,23 @@ RUN wget --no-verbose http://apache.mirrors.ovh.net/ftp.apache.org/dist/hive/hiv
 ENV HIVE_HOME=/apache-hive-1.2.2-bin
 ENV PATH=$HIVE_HOME/bin:$PATH
 
+#Kerberos libs
+ENV DEBIAN_FRONTEND noninteractive
+RUN DISTRO=debian && \
+    CODENAME=stretch && \
+    echo "deb [trusted=yes] http://repos.mesosphere.io/${DISTRO} ${CODENAME} main" > /etc/apt/sources.list.d/mesosphere.list && \
```

Comment:
Is adding the source repos.mesosphere.io necessary ?

Reply:
It's not indeed and it's been fixed